vcd/datasource_vcd_nsxt_network_dhcp_binding.go

```diff
@@ -0,0 +1,139 @@
+package vcd
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func datasourceVcdNsxtDhcpBinding() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdNsxtDhcpBindingRead,
+
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
How does `ForceNew` work in data sources? If I understood correctly, a change in `ForceNew` triggers a delete then create, but there is none of those in the data source, right?